src/i_winmusic.c

```diff
@@ -1555,15 +1517,12 @@ static void I_WIN_SetMusicVolume(int volume)
         // Ignore holding key down in volume menu.
         return;
     }
-
     last_volume = volume;
 
     volume_factor = sqrtf((float)volume / 15);
```

Comment:
`volume_factor` is also a shared resource. Should we move it to the critical section?

Reply:
Yes, good catch. Thanks.